app/gui/view/component-browser/src/lib.rs

```diff
@@ -87,7 +76,7 @@ impl Model {
     ) -> Vector2 {
         let half_node_height = NODE_HEIGHT / 2.0;
         let panel_left = -size.x / 2.0;
-        let panel_bottom = -size.y / 2.0;
+        let panel_bottom = 0.0;
```

Comment:
`panel_bottom` variable is now unnecessary

Reply:
I believe it contributes much for code readability.